packages/config/src/options.js

```diff
@@ -324,5 +324,20 @@ export function getNuxtConfig(_options) {
     bundleRenderer.runInNewContext = options.dev
   }
 
+  // Enable rate limit for dev mode
+  if (options.dev) {
+    if (typeof options.server.rateLimit === 'undefined') {
+      // It is unsual to request same resource more than 5 times in a 1sec!
+      options.server.rateLimit = {
+        windowMs: 1000,
+        max: 5,
+        keyGenerator: req => req.url,
```

Comment:
Is this in purpose that make rateLimit work for all clients connections instead of specific client ?

Reply:
Exactly, this is to avoid Safari loop requests on the same resource issue.